src/enclave/enclave_init.c

```diff
@@ -8,205 +8,202 @@
 #include "lkl/asm/host_ops.h"
 #include "lkl/setup.h"
 
+#define OE_BUILD_ENCLAVE
```

Comment:
What does this define do and why do we need it? Perhaps leave a comment?

Reply:
That tells the OE header files (attestation plugins) that we're building an enclave, which was a bug in OE that has been fixed. I think we can get rid of it once the latest OE has been merged.